### ISSUE TITLE
Cannot exclude PRs without labels.

### ISSUE THREAD

**Issue by oblogic7** [github-changelog-generator/github-changelog-generator#292]
None of my PRs have labels, but using the `--no-pr-wo-labels` flag has no effect on the PRs being included in the generated log.

```
{:tag1=>nil,
 :tag2=>nil,
 :date_format=>"%Y-%m-%d",
 :output=>"CHANGELOG.md",
 :base=>"HISTORY.md",
 :issues=>false,
 :add_issues_wo_labels=>true,
 :add_pr_wo_labels=>false,
 :pulls=>true,
 :filter_issues_by_milestone=>true,
 :author=>false,
 :unreleased=>true,
 :unreleased_label=>"Unreleased",
 :compare_link=>false,
 :enhancement_labels=>["enhancement", "Enhancement"],
 :bug_labels=>["bug", "Bug"],
 :exclude_labels=>
  ["duplicate",
   "question",
   "invalid",
   "wontfix",
   "Duplicate",
   "Question",
   "Invalid",
   "Wontfix"],
 :max_issues=>nil,
 :simple_list=>false,
 :verbose=>true,
 :header=>"# Change Log",
 :merge_prefix=>"**Merged pull requests:**",
 :issue_prefix=>"**Closed issues:**",
 :bug_prefix=>"**Fixed bugs:**",
 :enhancement_prefix=>"**Implemented enhancements:**",
 :git_remote=>"origin",
 :token=>"hidden value",
 :user=>"oblogic7",
 :project=>"secret"}
```

**Comment by olleolleolle** [comment]
It seems like that setting `add_issues_wo_labels` is merely parsed as an option, and never used in the code. Thanks for reporting this issue!

**Comment by mvz** [comment]
I think this was fixed in #618.
